### PR TITLE
feat: [IOPID-1499] Adopt the new DS layout to splash screens

### PR DESCRIPTION
**Pull request by shadowsheep1**
## Short description
This PR addresses the redesign of splash screens by adopting the new DS. 

> [!TIP]
> Review Privacy ✅
>
> [Run e2e tests](https://github.com/pagopa/io-app/actions/workflows/test-e2e.yml)

> [!NOTE]  
> On iOS the Splash Screen is so rapid, that VoiceOver doesn't have time to read the label.

## List of Changes
- Add the `ConstraintLayout` to the Android project to make life easier when building `XML` layouts.
- Use SVG files instead of PNG on both Android and iOS project. On Android the SVG files has been converted in `XML` [vector drawables](https://developer.android.com/develop/ui/views/graphics/vector-drawable-resources): all but `logo_consiglio_ministri` due to [this error](https://issuetracker.google.com/issues/37136187). The new Android splash screen has been tested on a real device running API level 23 to make sure that drawables were all good.

<details><summary>🎦 Demo 🎦</summary>
<p>

| Android API Level 23 | Android | iOS |
| - | - | - | 
|  |  |   |

| Android API Level 23 | Android a11y | iOS a11y |
| - | - | - | 
| <video src="https://github.com/pagopa/io-app/assets/16268789/34fa16a8-bdc3-4157-9c36-1e79cb8a6bd0" /> | <video src="https://github.com/pagopa/io-app/assets/16268789/b6a6439d-88af-4e0c-8135-1365fc4d5bdf" /> |  <video src="https://github.com/pagopa/io-app/assets/16268789/f07dd610-c282-482f-9eb5-a7620975e43c" /> |

</p>

</details> 

## How to test
- Do a fresh build on Android and iOS devices and check that splash screens have the new layout.

### CODE REVIEW

File: ios/ItaliaApp/LaunchScreen.storyboard
Comment (Ladirico) [inline ios/ItaliaApp/LaunchScreen.storyboard:1]: I run this PR on a physical device (iphone 13 mini) and the screen is slightly cut off to the side

iphone_13_visualization

Comment (shadowsheep1) [reply ios/ItaliaApp/LaunchScreen.storyboard:1]: Good catch! 

Fixed https://github.com/pagopa/io-app/pull/5554/commits/e31d5afffb401f1cd0a1329a7df07ef6d22e7031

Check it out now!

| New Splash |
| - |
| <video src="https://github.com/pagopa/io-app/assets/16268789/a25a8deb-25d8-421a-98e6-5e0dde83e7f8" /> |

Comment (Ladirico) [reply ios/ItaliaApp/LaunchScreen.storyboard:1]: perfect!

**Review by Ladirico — APPROVED** [review]
LGTM!

**Pull request merged**